ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -344,6 +349,8 @@ const Footer = ({
                     setStepNotification();
                 },
             });
+        } else if (activeStep.id === "accounts") {
+            onNext();
```

Comment:
Hm does this need special condition? The else { does the same.

Reply:
I guess you overlooked setUsers ?